### PR TITLE
Po

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**

### CODE REVIEW

File: Components/PO/alert.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.alert__container {
+  border: 1px solid var(--third--color);
+}
+
+.alert__all {
+  border: 1px solid gray;
```
Comment (j24m) [inline Components/PO/alert.css:6]: You need to use color variables everywhere no matter what.

**Review by j24m — COMMENTED** [review]
Great work , would just like to suggest you to add some margin-left to the  component-sidebar and to the right of the documentation. Use color variables instead of directly using color names or hex-codes. The rest seems good. Code is clean and readable to me .💯

File: Components/PO/alert.css (hunk 1 above)
Comment (ashishkatiyar2404) [reply Components/PO/alert.css:6]: ok. actually, the gray border will remove while doing documentation. That was on purpose.
Yup I will remember this.

**Comment by ashishkatiyar2404** [comment]
> Great work , would just like to suggest you to add some margin-left to the component-sidebar and to the right of the documentation. Use color variables instead of directly using color names or hex-codes. The rest seems good. Code is clean and readable to me .💯



> Great work , would just like to suggest you to add some margin-left to the component-sidebar and to the right of the documentation. Use color variables instead of directly using color names or hex-codes. The rest seems good. Code is clean and readable to me .💯



> Great work , would just like to suggest you to add some margin-left to the component-sidebar and to the right of the documentation. Use color variables instead of directly using color names or hex-codes. The rest seems good. Code is clean and readable to me .💯

Sure, Will remember your point😄

**Review by m26unkwn — APPROVED** [review]
Loved the BEM naming stye.

File: Components/PO/avatar.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  border-radius: 50%;
+}
+
+.avatar {
+  margin: 2rem;
+  border-radius: 50%;
+}
+
+.avatar-lg {
+  width: 200px;
```
Comment (dev-prabhat) [inline Components/PO/avatar.css:22]: try to use rem instead of px (greater than 10px)

File: style.css [hunk 3]
```diff
@@ -1 +1,101 @@
...
+  margin-left: 50rem;
+}
+
+.nav-home {
+  display: inline-block;
+  padding: 1rem;
+  background-color: var(--third--color);
+}
+
+.nav-home a {
```
Comment (dev-prabhat) [inline style.css:81]: use class to give styling to the tag

**Review by dev-prabhat — CHANGES_REQUESTED** [review]

File: Components/PO/alert.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+.alert {
+  width: 70%;
+  border-radius: 0.5rem;
+  margin: 1rem;
+  padding: 1rem;
+  font-size: 2rem;
+}
+
+.alert__primary {
+  color: #004085;
```
Comment (varun1233) [inline Components/PO/alert.css:21]: Try Using variables for color.

File: Components/PO/avatar.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
+.avatar__container {
+  border: 1px solid gray;
+}
+
+.avatar__all {
+  border: 1px solid gray;
+  margin: 2rem;
+  border-radius: 0.5rem;
```
Comment (varun1233) [inline Components/PO/avatar.css:8]: use px for values less than 1 rem

File: Components/PO/button.css [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+  border: 1px solid var(--btn-warning);
+}
+
+.btn__link-warning a {
+  color: var(--btn-warning);
+}
+
+/* icon button */
+.btn__icon-container {
+  /* border: 1px solid grey; */
```
Comment (varun1233) [inline Components/PO/button.css:102]: Remove commented code if not required

**Pull request merged**